### PR TITLE
Changed table and connetion string

### PR DESCRIPTION
**Pull request by Hinz3**
The table have multiple pages using DataTable.

### CODE REVIEW

File: Server/Views/Home/Search.cshtml [hunk 1]
```diff
@@ -1,39 +1,49 @@
...
-            </th>
+
             <th>
                 @Html.DisplayNameFor(model => model.LineNumber)
             </th>
             <th>
                 @Html.DisplayNameFor(model => model.LinePosition)
             </th>
+            <th>
+                @Html.DisplayNameFor(model => model.DocumentPath)
```
Comment (j2ghz) [inline Server/Views/Home/Search.cshtml:20]: Why do you want the path to be the last column? Isn't it the most important/interesting column?

**Review by j2ghz — CHANGES_REQUESTED** [review]
Good idea with datatables. We could also utilize https://datatables.net/manual/server-side later.

File: Server/Views/Home/Search.cshtml (hunk 1 above)
Comment (Hinz3) [reply Server/Views/Home/Search.cshtml:20]: The user dont have to see the path, they just need something to open the document. Show we actually dont need to show the path for them

Comment (j2ghz) [reply Server/Views/Home/Search.cshtml:20]: But they can't open the document yet. But ok.

**Review by j2ghz — APPROVED** [review]

**Pull request merged**